docs/procedures/rigid_waist_shift.md

```diff
@@ -0,0 +1,77 @@
+# LHC Local Coupling Corrections with a Rigid Waist Shift
+
+!!! note ""
+    Please keep in mind the [general checks](general_checks.md) for measurements.
+
+??? info "The Procedure in Short"
+
+      This methods aims to find the MQSX correction settings that would minimize betatron coupling and its impact on beam size *at IP*.
+      The method breaks the symmetry of the optics in the IR and forces local coupling RDTs to leak thoughout the machine, which makes them measurable through the $|C^{-}|$.
+
+
+      After global corrections are done and trimmed in the machine, one applies a rigid waist shift in a given IR and scans the colinearity knob for the value that minimises the $|C^{-}|$.
+      These settings, when taking away the rigid waist shift, will minimize local coupling and its impact at IP.
```

Comment:
minimise and minimize in consecutive sentences

Reply:
Resolved with [028c859](https://github.com/pylhc/pylhc.github.io/pull/71/commits/028c8596e86bf663e15301c7ca432ee1b6122333)